### ISSUE TITLE
Console throws an error

### ISSUE THREAD

**Issue by kgmahn** [mridoni/gix#134]
When running a cobol program from within GIX, the small console window displays the following error: Error opening terminal: unknown.
I am running Ubuntu 22.04.1 LTS with version Gix-IDE 1.1.1-9999. 

![gix-ide_console_Issue](https://user-images.githubusercontent.com/5904166/214458213-ef6ed707-97a0-41a4-a9ac-3ebc8ad40376.jpg)

**Comment by GitMensch** [comment]
To solve this: set the TERM variable, possibly to `xterm`. This _may_ be possible via the IDE settings (I don't know), otherwise do so on the terminal before starting the IDE from there. If it works you can add that to your system settings.

**Comment by mridoni** [comment]
> To solve this: set the TERM variable, possibly to `xterm`. This _may_ be possible via the IDE settings (I don't know), otherwise do so on the terminal before starting the IDE from there. If it works you can add that to your system settings.

Yes, that could solve it, unfortunately I was not able to reproduce this problem.
